book/src/drivers.md

```diff
@@ -27,6 +27,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Dell EMC Unity](https://github.com/dell/csi-unity) | `csi-unity.dellemc.com` | v1.0 | A Container Storage Interface (CSI) Driver for [Dell EMC Unity](https://www.dellemc.com/en-us/storage/unity.htm) | Persistent | Read/Write Single Pod | Yes | Snapshot
 [Dell EMC VxFlexOS](https://github.com/dell/csi-vxflexos) | `csi-vxflexos.dellemc.com` | v1.0 | A Container Storage Interface (CSI) Driver for Dell EMC VxFlexOS | Persistent | Read/Write Single Pod | Yes | Snapshot
 [Dell EMC XtremIO](https://github.com/dell/csi-xtremio-deploy) | `csi-xtremio.dellemc.com` | v1.0 | A Container Storage Interface (CSI) Driver for [Dell EMC XtremIO](https://www.dellemc.com/en-us/storage/xtremio-all-flash.htm) | Persistent | Read/Write Single Pod | Yes | Snapshot
+[Diamanti](https://diamanti.com/use-cases/container-network-and-storage-new/) | `dcx.csi.diamanti.com` | v1.1 | A Container Storage Interface (CSI) Driver for Diamanti Spektra | Persistent | Read/Write Single Pod | Yes | Mirroring, Replication, Encryption, Topology Aware, Snapshot, Expansion, Raw Block
```

Comment:
"Mirroring, Replication, Encryption" are not CSI features.  You can list them in your driver documentation, but this column is intended to show only CSI features.